docs/app-dev/subscribing-to-events-via-websocket.md

```diff
@@ -26,3 +26,34 @@ more information on query syntax and other options.
 You can also use tags, given you had included them into DeliverTx
 response, to query transaction results. See [Indexing
 transactions](./indexing-transactions.md) for details.
+
+### ValidatorSetUpdates
+
+When validator set changes, ValidatorSetUpdates event is published. The
+event carries a list of pubkey/power pairs. The list is the same
+Tendermint receives from ABCI application (see [EndBlock
+section](https://tendermint.com/docs/app-dev/abci-spec.html#endblock) in
+the ABCI spec).
+
+Response:
+
+```
+{
+    "jsonrpc": "2.0",
+    "id": "0#event",
+    "result": {
+        "query": "tm.event='ValidatorSetUpdates'",
+        "data": {
+            "type": "tendermint/event/ValidatorSetUpdates",
+            "value": {
+              "validator_updates": [
```

Comment:
@melekes does it contains only the new validators and validators that changed their `power` or the updated list with the new validator set ?

Reply:
Neither. It's the list of updates. So if the current validator set is A, B, C and application includes D, then `ValidatorSetUpdates` will contain only D.

Is that not enough? Given you track validators from the beginning (or get them using RPC `/validators`), you should be able to construct the updated set.

Reply:
And if the current validator set is A, B, C and and application includes D and drops A ? will `ValidatorSetUpdates` only include D as well ?

Reply:
No. It will be `A (pubkeyA, power: 0), D (pubkeyD, power: X)`